pkg/admin.go

```diff
@@ -119,7 +129,11 @@ func (r *Registry) VendorClaim(vendorID string, orgID string, orgName string, or
 		OrgKeys:          orgKeys,
 		Start:            time.Now(),
 	}, func(dataToBeSigned []byte, instant time.Time) ([]byte, error) {
-		return r.signAsOrganization(orgID, orgName, dataToBeSigned, instant)
+		if len(certificates) > 0 {
```

Comment:
could this check be in `signAsOrganization`? That would remove the double returns here.

Reply:
fixed!